pkg/nuctl/command/import.go

```diff
@@ -327,26 +317,65 @@ func (i *importProjectCommandeer) importProject(projectConfig *ProjectImportConf
 	}
 
 	functionImportErr := i.importFunctions(projectConfig.Functions, projectConfig.Project.Meta.Name)
-	functionEventImportErr := i.importFunctionEvents(projectConfig.FunctionEvents)
-
 	if functionImportErr != nil {
-		return errors.Wrap(functionImportErr, "Failed to import some functions")
+		i.rootCommandeer.loggerInstance.WarnWith("Unable to import all functions",
+			"functionImportErr", functionImportErr)
+
+		// return this error
+		err = functionImportErr
 	}
+
+	functionEventImportErr := i.importFunctionEvents(projectConfig.FunctionEvents)
 	if functionEventImportErr != nil {
-		return errors.Wrap(functionImportErr, "Failed to import some function events")
+		i.rootCommandeer.loggerInstance.WarnWith("Unable to import all function events",
+			"functionEventImportErr", functionEventImportErr)
+
+		// return this err only if not previously set
+		if err == nil {
+			err = functionEventImportErr
+		}
 	}
 
-	return nil
+	return err
 }
 
 func (i *importProjectCommandeer) importProjects(projectImportConfigs map[string]*ProjectImportConfig) error {
-	var g errgroup.Group
+	i.rootCommandeer.loggerInstance.DebugWith("Importing projects", "projects", projectImportConfigs)
 
+	// TODO: parallel this with errorGroup, mutex is required due to multi map writers
```

Comment:
In the `ProjectImportConfig` (line 199), you changed the `Project` member to be a pointer, which forces the project import to be serial.

Reply:
explained offline as well:
> when you unmarshal to a non-pointer struct, it will be hard to distinguish between an invalid unmarshaling to valid but does not match the structure.

To make it right, we must use pointer to struct (e.g. Project), so once unmarshalled I would have the right tools to distinguish between _multi_ projects and a _single_ project